src/screens/AdminPortal/OrgList/modal/OrganizationModal.tsx

```
@@ -37,27 +39,23 @@
  * @returns The rendered organization modal.
  */
 import React, { type ChangeEvent } from 'react';
-import Button from 'shared-components/Button';
+import Button from 'react-bootstrap/Button';
```

ESLint

[error] 'react-bootstrap/Button' import is restricted from being used. Direct imports of react-bootstrap/Button are not allowed. Use the shared Button component from src/shared-components/Button/ instead.

(no-restricted-imports)